js/src/functions.js

```diff
@@ -164,8 +164,8 @@ Functions.addDatepicker = function ($thisElement, type, options) {
                 var tooltip = $thisElement.uiTooltip('instance');
                 if (typeof tooltip !== 'undefined') {
                     tooltip.disable();
-                    var $note = $('<p class="note"></div>');
-                    $note.text(tooltip.option('content'));
+                    var $note = $('<p class="note" style="font-size: 11px;"></div>');
+                    $note.text(Messages.strNote + ': ' + tooltip.option('content'));
```

Comment:
```suggestion
                    $note.text(Messages.strNote + tooltip.option('content'));
```
Note: `': '` should go to strNote because some languages space it differently.
I am copying this change into 5.3 because it's an enhancement and it requires a translation to be done